example_code/applications/multi_call/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "stylus-multi-call-contract"
+version = "0.1.5"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+description = "Stylus multi call example"
+
+[dependencies]
+alloy-primitives = "0.3.1"
```

Comment:
With the latest cargo stylus, this config does not compile. But if we change allow-primites and allow-sol-types to 0.7.6 and the stylus-sdk to 0.5.2 it builds correctly.

If it's the same case for you, can you update the config here?